### ISSUE TITLE
Use 2 data entry form templates for projects (NPMS type & non-NPMS type)

### ISSUE THREAD

**Issue by andrewvanbreda** [BiologicalRecordsCentre/plantportal#3]
When setting up a a project/survey in Plant Portal, the administrator needs to be able to select one of 2 types of data entry form for the project. 
One type is similar to NPMS, the other type is different to NPMS.
These 2 templates will use their own standard structure for the data, such as the attributes.

**Comment by andrewvanbreda** [comment]
@sacrevert 
Hi Oli, 

A couple of questions:

1. Is the abundance count going to be a number (like Wildflower), or a drop-down?
2. Whenever you are ready, will need a list of attributes for the non-NPMS type, no rush from my side.

Most of the NPMS type is setup part from the abundance and plot.

**Comment by sacrevert** [comment]
Hi @andrewvanbreda 

1. I assume that you are asking here about the NPMS form (rather than non-NPMS)? Given that we have simplified the form relative to the true NPMS website, in that now the level (wildflower/indicator/inventory) is a drop-down choice rather than a separate form, then I think the simplest is for the abundance count to be as per the indicator/inventory route on the true NPMS website. I suggest that, at least in the first instance, we have the taxon-list restriction that we have for the indicator form apply to both the wildflower and indicator choice on this website. This means that: (a) actually the user will have the option of choosing more species than they actually should (i.e. indicator species will be available to those recording at indicator level; and (b) the species page of the form will need to respond to which level the user chooses (i.e. the restricted indicator list will be used if the user selects wildflower or indicator, where as the restrictions that we have in place for inventory level will apply if they choose inventory (if i recall, we have a list of higher level taxon groups that the user is restricted to at inventory level on the NPMS website).
2. OK, am planning to do that now.
3. Just in terms of anywhere this might be relevant on the website GUI, can we call non-NPMS "standard mode" instead of non-NPMS from now on? So we have NPMS mode and standard mode.
Cheers!

EDIT
Regarding (1), I suppose we do actually have the wildflower list tagged as such in the database because of the specific form on the true NPMS website. If this is the case, then can we have the available species list being reactive to all three level choices please?

**Comment by sacrevert** [comment]
@andrewvanbreda @DavidRoy 
Andy, here is my outline for the data entry form templates for standard mode. There is a fair bit to accommodate for the general sample page, so I wonder whether we could group these into expandable sections, the less important ones being collapsed by default. Let me know where you have questions -- would be happy to have a web meeting if anything needs discussing.
[standardPpFields_v0.docx](https://github.com/BiologicalRecordsCentre/plantportal/files/5990606/standardPpFields_v0.docx)

**Comment by andrewvanbreda** [comment]
@sacrevert 
Hi Oli, Thanks for this info. Lots of detail here, so I will go through what you say and let you know

**Comment by andrewvanbreda** [comment]
@sacrevert 
Hi Oli,

Yes It was NPMS type I meant with that question I think. I have made the abundance a drop-down now which is also consistent with Tracking The Impact which is what we were basing the form on.

Yes, Standard Mode is good, that is better name, we'll go with that.

In terms of some of the other things you have written here, I think I will raise some new issues and close this one, as there are too many things for one issue now I think.
The original purpose of this issue was simply allow support for two different types of project forms, which I believe we now have in the general sense.

By the way, I don't usually get involved with Github maintenance, but I added a "Coding complete" label this morning for this project.
